.gitlab-ci.yml

```
@@ -101,7 +101,14 @@ publish:tests:
   dependencies:
     - test:unit
   script:
-    - npx codecov
+    - export CI_BUILD_REF=${CI_COMMIT_SHA}
+    - export CI_BUILD_REF_NAME=${CI_COMMIT_REF_NAME}
+    - export CI_BUILD_NAME=${CI_JOB_NAME}
+    - export CI_BUILD_ID=${CI_JOB_ID}
+    - export CI_MERGE_REQUEST_IID=${CI_COMMIT_BRANCH#pr_}
```

Shouldn't it be:
```suggestion
    - export CI_MERGE_REQUEST_ID=${CI_COMMIT_BRANCH#pr_}
```
?

Anyway, this won't be correctly used as we do GitHub PRs and not GitLab MRs

---

the `IID` is read in the coveralls dependency -> https://github.com/nickmerwin/node-coveralls/blob/master/lib/getOptions.js#L90 - and I _think_ this is used to determine the PR this is coming from, but I'll re-run without and see if the information available on coveralls is the same

---

It's okey. We can leave it